build.gradle

```diff
@@ -76,7 +76,7 @@ subprojects {
                 limit {
                     counter = 'METHOD'
                     value = 'COVEREDRATIO'
-                    minimum = 1.00
+                    minimum = 0.98
```

Comment:
I am not sure if it is a good idea but maybe a comment here would help us know why we need to lower the test coverage and in the future revisit the cases that are difficult to test. Maybe a task in JIRA can be even better?